setup.sh

```diff
@@ -34,6 +34,17 @@ export PYTHONPATH=$REPO_PATH/python:$PYTHONPATH
 echo "Setting ASSASSYN_HOME to $REPO_PATH"
 export ASSASSYN_HOME=$REPO_PATH
 
+# Add PyCDE specific PYTHONPATH conditionally
+# As per review feedback: Check if PyCDE is importable before adding its path.
+echo "Checking for PyCDE installation..."
+python -c "import pycde" &> /dev/null 
+if [ $? -eq 0 ]; then 
```

Comment:
Isn't this wrong? Think about this:
If you have `$? == 0` in script `init.sh`, you will NOT build circt.
However, you load circt PyCDE here.

The correct thing is `$? == 0`, you should do nothing. PyCDE is already there. Ow, you just assume circt is built in the 3rd-party path.